### ISSUE TITLE
Checking is mount does not work correctly with "bind" option

### ISSUE THREAD

**Issue by piotrplenik** [ansible/ansible-modules-core#2924]
**Issue Type:**

Bug Report

**Ansible Version:**

ansible-modules-core devel

**Environment:**
Ansible is running from Ubuntu 15.04
I am managing an Debian Wheezy instance

**Summary:**

When using the mount module and specifying an option of "bind", checking is mountpoint exist does not work properly. This has been described in #86. 

In this issue, I replaced `os.path.ismount(path)` into separated method, that can recognize bind too.

**Comment by ansibot** [comment]
@grossws ping, this issue is waiting for your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@grossws, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by abadger** [comment]
Closing as duplicate of #1942.  The changes proposed by @jupeter have largely been merged to stable-2.1 and devel now.
